### ISSUE TITLE
Range 10 needed on multiple fights.

### ISSUE THREAD

**Issue by SaintEcho** [DeadlyBossMods/DBM-Classic#673]
Four Horsemen you need range 10 for zeliak
saph you need range 10 for air phase
kt everyone needs it for frost blast all of p2/p3 (pretty sure it only shows for detonate mana)

Theres no option to show it so you have to manually type it but if you type it before the pull it disappears automatically and you have to redo it mid pull.

**Comment by MysticalOS** [comment]
KT shows it entirety of phase 2 and never auto closes until fight is over.

it also shouldn't hide if it was manully shown before pull, what boss is doing that?

**Comment by hauk92** [comment]
Range 10 after the Thaddius fight till the debuff ticks out would be nice, too.

**Comment by wallrik** [comment]
My take on this issue:
* **Sir Zeliek:** I can't think of any *reliable* way to continuously check if you are within 35y of the boss, and only show the range frame for that. The way most people play this fight is that range stays 35 yards away as to avoid the beam completely, which makes the range check irrelevant. My take on the feature request is that it's unnecessary.
* **Sapphiron:** Range frame was added in 7fd01059f4287fa6b0be58b92c663c349d0ddf9b (version 1.13.69)
* **Kel'Thuzad:** Auto-showing the range frame for P2 was fixed in c9bd44d0697c74a8d64cd28f49a1193af0de218c (version 1.13.68)
* **Thaddius:** I looked into adding this feature, but turns out the range is [13 yards](https://classic.wowhead.com/spell=28085/negative-charge), and DBM unfortunately cannot show that range in raids. There's a deadzone over 11y and under 18y, so the range check would have to be 18y to be safe, which is far too large, in my opinion.
